docs/proposed/http-storage-node-protocol.rst

```diff
@@ -380,6 +380,11 @@ then the expiration time of that lease will be changed to 31 days after the time
 If it does not match an existing lease
 then a new lease will be created with this ``renew-secret`` which expires 31 days after the time of this operation.
 
+The renew and cancel secrets must be 32 bytes long
+(or in the case of JSON encoding they must UTF-8 encode to 32 bytes).
+The server treats them as opaque values.
+:ref:`Share Leases` gives details about how the Tahoe-LAFS storage client constructs these values.
```

Comment:
It doesn't cover cancel secrets... You could add a bit about cancel secret generation, or you could add a bit about how "cancellation is not actually supported in practice."